lib/tasks/update_tmdb_data.rake

```diff
@@ -0,0 +1,28 @@
+# To run the rake task: rake "tmdb_data:refresh"
+# To run from console: Rake::Task["tmdb_data:refresh"].invoke
+
+TIME_FRAME = (ENV['tmdb_refresh_time_frame_days'] || 7).to_i.days.ago
+BATCH_SIZE = (ENV['tmdb_refresh_batch_size'] || 50).to_i
+
+namespace :tmdb_data do
+  desc 'refresh tmdb_data'
+  task :refresh => :environment do
+    tmdb_ids = Movie
+      .where('updated_at < ?', TIME_FRAME)
+      .order(updated_at: :asc)
+      .limit(BATCH_SIZE)
+      .pluck(:tmdb_id)
+
+    puts "*** Refreshing #{tmdb_ids.count} movies..."
+    updated_movies = []
+
+    tmdb_ids.each do |tmdb_id|
+      TmdbHandler.tmdb_handler_update_movie(tmdb_id)
+      puts "Movie refreshed. tmdb_id: #{tmdb_id}"
+      updated_movies << tmdb_id
```

Comment:
From what I can tell, the `updated_movies` count will always be the same as the input count. There aren't any conditions in this loop that would prevent a movie from being counted. When we raise an error, it just stops the loop, but doesn't adjust the output count. 

I'm not prioritizing making this count smarter. It seems like it could be a lot of effort for minimal reward.